dev/travis/before_script.sh

```diff
@@ -76,4 +76,20 @@ case $TEST_SUITE in
 
         cd ../../..
         ;;
+    js)
+        curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.1/install.sh | bash
+        export NVM_DIR="$HOME/.nvm"
+        [ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh" # This loads nvm
+        nvm install --lts
```

Comment:
You are always up to date, don't have to follow releases schedule etc. When new LTS version will be up, your tests will start using it.

But to just quit discussion, I'll change it to `6` and this will be good enough till next LTS.

Docs about `engines` option from `package.json`
> this field is advisory only will produce warnings when your package is installed as a dependency.

So in our case, we can simply remove this param.